js/src/app/dashboard/_components/RecentSubmissions/SubmissionDetails.tsx

```diff
@@ -73,7 +73,7 @@ export default function SubmissionDetails() {
           <Text size="lg">{acceptanceRate * 100}%</Text>
         </div>
         <Card shadow="xs" padding="lg" radius="lg" mt="xl">
-          <div dangerouslySetInnerHTML={{ __html: description }}></div>
+          <div dangerouslySetInnerHTML={{ __html: description ?? "" }}></div>
```

Comment:
Let's make the nullish operator's default value more explicit. In this case, we can put "No available description found."

```suggestion
          <div dangerouslySetInnerHTML={{ __html: description ?? "No available description found." }}></div>
```